packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.2.0]
+
+### Added
+
+- Add export for const `SUPPORTED_CHAIN_IDS` in `token-prices-service.ts` and `index.ts` ([#4079](https://github.com/MetaMask/core/pull/4079))
```

Comment:
The only relevant user-facing detail here is that a package-level export was added, so we don't need specific file names. We could add a bit more context for readers of the changelog:

```suggestion
- `CodefiTokenPricesServiceV2` exports `SUPPORTED_CHAIN_IDS`, an array of chain IDs supported by Codefi Price API V2. ([#4079](https://github.com/MetaMask/core/pull/4079))
```